packages/react-api/src/hooks/useTileFeatures.js

```diff
@@ -120,7 +120,7 @@ export default function useTileFeatures({
   );
 
   const onDataLoad = useCallback(({ tiles: [tile] }) => {
-    const tilesFormat = new URLSearchParams(tile).get('formatTiles');
```

Comment:
I didn't know this could fail on that circumstance

Reply:
for example (new URLSearchParams('https://github.com?formatTiles=aaaa')).get('formatTiles')